authors/urls.py

```diff
@@ -19,6 +19,9 @@
 urlpatterns = [
     path('', include('authors.apps.articles.url')),
     path('admin/', admin.site.urls),
-    path('api-auth/', include('rest_framework.urls', namespace='rest_framework')),
-    path('api/', include(('authors.apps.authentication.urls', 'authentication'), namespace='authentication')),
+    path('api-auth/', include(
+        'rest_framework.urls', namespace='rest_framework')),
+    path('', include('authors.apps.articles.url')),
+    path('api/profiles/', include('authors.apps.profiles.urls')),
+    path('api/', include('authors.apps.authentication.urls')),
```

Comment:
@leonardnjura Ensure we have the same naming of the urls. Liaise with the team on this.